src/clients/api/queries/getHypotheticalAccountLiquidity.ts

```diff
@@ -0,0 +1,37 @@
+import BigNumber from 'bignumber.js';
+import { Contract } from 'web3-eth-contract';
+import { Asset } from 'types';
+
+export interface IGetHypotheticalAccountLiquidityInput {
+  comptrollerContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | undefined | null;
```

Comment:
`account` should be mandatory here.

Reply:
It is mandatory. but it can be undefined or null